compact.go

```diff
@@ -774,12 +780,23 @@ func (c *LeveledCompactor) populateBlock(blocks []BlockReader, meta *BlockMeta,
 				}
 
 				it := &deletedIterator{it: chk.Chunk.Iterator(), intervals: dranges}
+
+				var (
+					min = int64(math.MinInt64)
+					t   int64
+					v   float64
+				)
 				for it.Next() {
```

Comment:
Can we check `it.Err` just to follow good patterns (even if current implementaiton does return error)